reconfiguration_action_name: 'rosgraph_manipulator_action_server'
configurations:
  f1_v1_r1:
    command: roslaunch f1_v1_r1 f1_v1_r1.launch
  f1_v1_r2:
    command: roslaunch f1_v1_r2 f1_v1_r2.launch
  f1_v1_r3:
    command: roslaunch f1_v1_r3 f1_v1_r3.launch
  f1_v2_r1:
    command: roslaunch f1_v2_r1 f1_v2_r1.launch
  f1_v2_r2:
    command: roslaunch f1_v2_r2 f1_v2_r2.launch
  f1_v2_r3:
    command: roslaunch f1_v2_r3 f1_v2_r3.launch
  f1_v3_r1:
    command: roslaunch f1_v3_r1 f1_v3_r1.launch
  f1_v3_r2:
    command: roslaunch f1_v3_r2 f1_v3_r2.launch
  f1_v3_r3:
    command: roslaunch f1_v3_r3 f1_v3_r3.launch
  f2_v1_r1:
    command: roslaunch f2_v1_r1 f2_v1_r1.launch
  f2_v1_r2:
    command: roslaunch f2_v1_r2 f2_v1_r2.launch
  f2_v1_r3:
    command: roslaunch f2_v1_r3 f2_v1_r3.launch
  f2_v2_r1:
    command: roslaunch f2_v2_r1 f2_v2_r1.launch
  f2_v2_r2:
    command: roslaunch f2_v2_r2 f2_v2_r2.launch
  f2_v2_r3:
    command: roslaunch f2_v2_r3 f2_v2_r3.launch
  f2_v3_r1:
    command: roslaunch f2_v3_r1 f2_v3_r1.launch
  f2_v3_r2:
    command: roslaunch f2_v3_r2 f2_v3_r2.launch
  f2_v3_r3:
    command: roslaunch f2_v3_r3 f2_v3_r3.launch
  f3_v1_r1:
    command: roslaunch f3_v1_r1 f3_v1_r1.launch
  f3_v1_r2:
    command: roslaunch f3_v1_r2 f3_v1_r2.launch
  f3_v1_r3:
    command: roslaunch f3_v1_r3 f3_v1_r3.launch
  f3_v2_r1:
    command: roslaunch f3_v2_r1 f3_v2_r1.launch
  f3_v2_r2:
    command: roslaunch f3_v2_r2 f3_v2_r2.launch
  f3_v2_r3:
    command: roslaunch f3_v2_r3 f3_v2_r3.launch
  f3_v3_r1:
    command: roslaunch f3_v3_r1 f3_v3_r1.launch
  f3_v3_r2:
    command: roslaunch f3_v3_r2 f3_v3_r2.launch
  f3_v3_r3:
    command: roslaunch f3_v3_r3 f3_v3_r3.launch
  f1_v1_r3_k:
    command: roslaunch f1_v1_r3_k f1_v1_r3_k.launch
  f1_v3_r1_k:
    command: roslaunch f1_v3_r1_k f1_v3_r1_k.launch
  f2_v1_r3_k:
    command: roslaunch f2_v1_r3_k f2_v1_r3_k.launch


kill_nodes: ['/move_base']
save_action: 'move_base'
goal_msg_type: move_base_msgs.msg.MoveBaseAction

